src/components/FindMovie/FindMovie.js

```diff
@@ -1,55 +1,116 @@
-import React from 'react';
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import cn from 'classnames';
+
 import './FindMovie.scss';
 
 import { MovieCard } from '../MovieCard';
-import movies from '../../api/movies.json';
-
-export const FindMovie = () => (
-  <>
-    <form className="find-movie">
-      <div className="field">
-        <label className="label" htmlFor="movie-title">
-          Movie title
-        </label>
-
-        <div className="control">
-          <input
-            type="text"
-            id="movie-title"
-            placeholder="Enter a title to search"
-            className="input is-danger"
-          />
-        </div>
+import { getMovie } from '../../api/api';
 
-        <p className="help is-danger">
-          Can&apos;t find a movie with such a title
-        </p>
-      </div>
+export const FindMovie = ({ addMovie }) => {
+  const [movie, setMovie] = useState(null);
+  const [title, setTitle] = useState('');
+  const [isError, setIsError] = useState(false);
+
+  const findMovie = () => {
+    getMovie(title)
+      .then((data) => {
+        if (data.Response === 'False') {
+          setIsError(true);
+
+          return;
+        }
+
+        setMovie({
+          title: data.Title,
+          description: data.Plot,
+          imgUrl: data.Poster,
+          imdbUrl: `https://www.imdb.com/title/${data.imdbID}/`,
+          imdbId: data.imdbID,
+        });
+      });
+  };
+
+  return (
+    <>
+      <form className="find-movie">
+        <div className="field">
+          <label className="label" htmlFor="movie-title">
+            Movie title
+          </label>
 
-      <div className="field is-grouped">
-        <div className="control">
-          <button
-            type="button"
-            className="button is-light"
-          >
-            Find a movie
-          </button>
+          <div className="control">
+            <input
+              className={cn('input', { 'is-danger': isError })}
+              type="text"
+              id="movie-title"
+              placeholder="Enter a title to search"
+              value={title}
+              onChange={(event) => {
+                setTitle(event.target.value);
+                setMovie(null);
+                setIsError(false);
+              }}
+            />
+          </div>
+
+          {isError
+            && (
+              <p className="help is-danger">
+                Can&apos;t find a movie with such a title
+              </p>
+            )
+          }
         </div>
 
-        <div className="control">
-          <button
-            type="button"
-            className="button is-primary"
-          >
-            Add to the list
-          </button>
+        <div className="field is-grouped">
+          <div className="control">
+            <button
+              className="button is-light"
+              type="button"
```

Comment:
You can make this button of type `submit` and call the function `findMovie` on the form submit
(`<form onSubmit={findMovie}>`)
In such way you can handle pressing the `Enter` button after entering the search query